src/main/java/forestry/core/recipes/RecipeUtil.java

```diff
@@ -141,6 +143,10 @@ public static void addRecipe(String recipeName, Item item, Object... obj) {
 	}
 
 	public static void addRecipe(String recipeName, ItemStack itemstack, Object... obj) {
+		if (Config.isDebug && itemstack.getMetadata() > 15) {
```

Comment:
Items can have metadatas higher than 15. Only blocks can't have them.

Reply:
OK, I have removed this and the unnecessary commits